### PR TITLE
getopts for command line argument handling + custom font options

### PR DESCRIPTION
**Pull request by ohno9119**
The application now uses getopts for argument handling, this should make adding extra command line options less painful in the future.
This PR also adds options for customizing the font used. (Workaround for issue #44).

### CODE REVIEW

File: src/activate_linux.c [hunk 1]
```diff
@@ -25,6 +26,21 @@ void draw(cairo_t *cr, char *title, char *subtitle, float scale, struct rgba_col
 
     // set font size, and scale up or down
     cairo_set_font_size(cr, 24 * scale);
+    
+    // font weight and slant settings 
+    cairo_font_weight_t font_weight = CAIRO_FONT_WEIGHT_NORMAL;
+    if (boldmode == 1) {
+	font_weight = CAIRO_FONT_WEIGHT_BOLD;
```
Comment (ReperakDev) [inline src/activate_linux.c:33]: Not entirely sure what this formatting is here.

File: src/activate_linux.c [hunk 2]
```diff
@@ -68,61 +84,47 @@ int main(int argc, char *argv[]) {
...
+    int opt;
+    while ((opt = getopt(argc, argv, "?bit:m:s:f:c:")) != -1) {
+           switch (opt) {
+           case 'b':
+               boldmode = 1;
+               break;
+           case 'i':
+               slantmode = 1;
+               break;
+		   case 't':
```
Comment (ReperakDev) [inline src/activate_linux.c:105]: Space is inconsistent here.

File: src/activate_linux.c [hunk 3]
```diff
@@ -68,61 +84,47 @@ int main(int argc, char *argv[]) {
...
+               break;
+		   case 'f':
+               customfont = optarg;
+               break;
+		   case 's':
+               scale = atof(optarg);
+               break;
+		   case 'c':
+                text_color = rgba_color_string(optarg);
+				if (text_color.a < 0.0) {
```
Comment (ReperakDev) [inline src/activate_linux.c:119]: Space is inconsistent here.

**Review by ReperakDev — CHANGES_REQUESTED** [review]

**Comment by MrGlockenspiel** [comment]
Wow you did my job for me
I’m not particularly worried about the formatting, that can always be fixed later
Thanks 👍

**Pull request merged**